feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SessionsStrings.kt

```diff
@@ -72,6 +74,7 @@ sealed class SessionsStrings : Strings<SessionsStrings>(Bindings) {
                 UserIcon -> "User icon"
                 EventDay -> "Day"
                 Category -> "Category"
+                SearchPlaceHolder -> "Enter some technology"
```

Comment:
Placeholder can't display many characters, so I made it simple. Please let me know if any corrections are needed.

Reply:
👍